packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/IJSII417Derived.java

```diff
@@ -11,23 +11,27 @@
      */
     @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
     java.lang.String getProperty();
```

Comment:
"get" here but not on others?

Reply:
In this 'example', bar() and baz() are methods, not properties. The get prefix has only been applied to properties from the original typescript.